apps/web-client/src/lib/components/Page.svelte

```diff
@@ -76,9 +76,9 @@
 		},
 		{
 			label: tNav.supplier_orders(),
-			href: appPath("suppliers"),
+			href: appPath("supplier_orders"),
 			icon: Truck,
-			linkto: "orders/suppliers"
+			linkto: "orders/suppliers/orders"
```

Comment:
We also need a way to be able to reach `/orders/suppliers` (supplier management page)